detail.py

```diff
@@ -0,0 +1,25 @@
+from django.conf import settings
+from django_mako_plus import view_function, jscontext
+from datetime import datetime, timezone
+from catalog import models as cmod
+
```

Comment:
Think about deleting imports that you didn't use or need.

Reply:
That's a good idea! I will take a look at them.